Mathlib/RingTheory/Ideal/KrullDimension.lean

```diff
@@ -0,0 +1,100 @@
+/-
```

Comment:
I don't really see why this file is in the `Ideal` subdirectory, probably it should just be `RingTheory/KrullDimension.lean`.